test/integration/cmn/template/channel_transfer_template.go

```diff
@@ -23,7 +23,11 @@ options:
   newestRequestStreamBufferSize: 50
 channels:{{ range .Channels }}
   {{- if ne . "acl" }}
-  - {{ . }}
+  - name: {{ . }}
+    {{- if $.HasBatcher }}
+    batcher:
+      addressGRPC: "{{ $.BatcherGRPCAddress }}"
+    {{- end }}
```

Comment:
How can I set channels working with and without butcher at the same time?

Reply:
I think it will complicate the configuration of tests. you will have to specify other parameters for each channel in addition to the name. and you need to use the batcher only for one test in the channel transfer. 

If necessary, I can try to offer a variant of such configuration

Reply:
1. I need to check how the transfer will work in a mixed environment.
1. I think we need to change the channel section

Reply:
Moved batcher config from network to channels